stonesoup/movable/actionable.py

```diff
@@ -0,0 +1,9 @@
+from ..sensormanager.actionable import Actionable
+
+
+class MovableActionable(Actionable):
+    """Movable Actionable base class
+
+    Base class for actionable sensors which has all the functionality of base
+    :class:`~.Actionable` class, with additional platform-specific features.
+    """
```

Comment:
I don't love that we now have `movable.actionable` and `movable.actionable_movable` modules... 

I think it makes sense in terms of the code because this class is the `MovableActionable` base class whereas the other one is a `Movable` which has some of that actionable functionality but the naming is a bit confusing. Would renaming the `actionable_movable` module be too much of a headache @sdhiscocks?

Reply:
It also needs adding to stonesoup.movable.rst so it appears in the docs